File: graphql-test/src/main/java/com/junyss/graphqltest/api/people/model/dto/request/PeopleUpdateRequestDto.java
package com.junyss.graphqltest.api.people.model.dto.request;

import com.junyss.graphqltest.common.enumtype.BloodType;
import com.junyss.graphqltest.common.enumtype.Role;
import com.junyss.graphqltest.common.enumtype.Sex;

import lombok.Data;

@Data
public class PeopleUpdateRequestDto {
	private Long peopleId;
	private Long teamId;
	private String lastName;
	private String firstName;
	private Sex sex;
	private BloodType bloodType;
	private Integer serveYears;
    private Role role;
	private String hometown;
}

File: kotlin-graphql-test/src/main/kotlin/com/junyharangstudy/kotlingraphqltest/api/people/model/dto/request/PeopleUpdateRequestDto.kt
package com.junyharangstudy.kotlingraphqltest.api.people.model.dto.request

import com.junyharangstudy.kotlingraphqltest.api.common.enumtype.BloodType
import com.junyharangstudy.kotlingraphqltest.api.common.enumtype.Role
import com.junyharangstudy.kotlingraphqltest.api.common.enumtype.Sex

class PeopleUpdateRequestDto (
    var teamId: Long?,
    var lastName: String?,
    var firstName: String?,
    var sex: Sex?,
    var bloodType: BloodType?,
    var serveYears: Int?,
    var role: Role?,
    var hometown: String?
)